airflow/templates/pvc.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if and .Values.airflow.persistence.enabled (not .Values.airflow.persistence.existingClaim) }}
```

Comment:
arg, that's how we do a `and` in helm templates? That sucks! Why don't they use jinja?

Reply:
I know. Luckily I pinched this example from the mysql helm chart. The go templating language has been a little tricky at times